omero/util_scripts/Channel_Offsets.py

```diff
@@ -360,11 +360,13 @@ def run_script():
 
     try:
         script_params = client.getInputs(unwrap=True)
+        print(script_params)
```

Comment:
Did you intend for these to stay?

Reply:
Yes, I think it's generally useful to have some logging of the params (and other info) when you ran a script.